networking/synthetic/stac-on-k8s/ReadMe.md

```diff
@@ -0,0 +1,52 @@
+Run STAC-N1 on k8s deployment
+===========================
+
+Scripts and automation to run containerized STAC-N1 inside k8s pods
+leveraging [SFC device plugin](https://github.com/vikaschoudhary16/kubernetes/pull/7).
+
+----------
+
+**Pre-requisites:**
+-----------------------
+- K8s 1.8 or higher patched with the [PR for device plugin's extended support for smart-NICs](https://github.com/kubernetes/kubernetes/pull/51938)
+- Onload 1.3+ installed and working on the hosts.
+
+**Steps:**
+------
+
+ **Deploy SFC device plugin daemonset pods:**
```

Comment:
what about all the export commands ?

Reply:
I have to run this to successfully deploy the sfc:
"kubectl config set-context $(kubectl config current-context) --namespace=kube-system"